libpam/pam_start.c

```diff
@@ -25,6 +25,21 @@ static int _pam_start_internal (
     D(("called pam_start: [%s] [%s] [%p] [%p]"
        ,service_name, user, pam_conversation, pamh));
 
+    /* Bind text domain to pull in PAM transpations for a case where
+       linux-pam is installed to non-default prefix.
+
+       It is safe to call bindtextdomain() from multiple threads, but it
+       has a chance to have some overhead. Let's try to do it once (or a
+       small number of times as `bound_text_domain` is not protected by
+       a lock. */
+    static int bound_text_domain = 0;
+    if (!bound_text_domain) {
+	bound_text_domain = 1;
+#ifdef HAVE_BINDTEXTDOMAIN
+	bindtextdomain(PACKAGE, LOCALEDIR);
+#endif
+    }
```

Comment:
Let's move the whole hunk under `#ifdef HAVE_BINDTEXTDOMAIN`.

Reply:
Moved all under `#ifdef HAVE_BINDTEXTDOMAIN`